about-us.html

```diff
@@ -302,7 +302,7 @@ <h3 class="main-content-heading fs">SUBSCRIBE</h3>
                         <input class="email-input" name="email" type="email" placeholder="Enter your email"
                             pattern="[a-z0-9._%+\-]+@[a-z0-9.\-]+\.[a-z]{2,}$">
                         <label class="email-label" for="email"></label>

```

Comment:
Inside your labels, add text and hide it using a visually-hidden class. This is important for accessibility, ensuring that screen readers can properly interpret and navigate the form elements.